wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md

```diff
@@ -10,12 +13,12 @@ There are 2 types of compression. **Lossy** and **Lossless** compression.
 Zip
 ---
 
-The \*.osz package is actually a \*.zip file, and will compress your beatmap, although not very much, since audio, video, and images are usually compressed very well. Zip and other archives are all lossless compression, otherwise you wouldn't be able to get your data back.
+The `*.osz` package is actually a `*.zip` file, and will compress your beatmap, although not very much, since audio, video, and images are usually compressed very well. Zip and other archives are all lossless compression, otherwise you wouldn't be able to get your data back.
```

Comment:
idk, when I wrote the ASG, someone (I don't remember who but they had) suggested that we don't prefix file ext with an `*` (and so that became a thing)